<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEdgeDialog</class>
 <widget class="QDialog" name="EditEdgeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Edge</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons.qrc">:/icons/edge_edit.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_2">
        <property name="styleSheet">
         <string notr="true">QLabel { font-size: 14px; }</string>
        </property>
        <property name="text">
         <string>Edit Edge</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="dialogMainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Edge Details</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="idLabel">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="idEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelLabel">
           <property name="text">
            <string>Label</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="labelEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="fromLabel">
           <property name="text">
            <string>From</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="toLabel">
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelMark">
           <property name="text">
            <string>Mark</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="markComboBox"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="fromNode">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="toNode">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="bidirectionalBox">
           <property name="text">
            <string>Bidirectional</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditEdgeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditEdgeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
